Update RunningClimateMachine.md (#1325)
CliMA/ClimateMachine.jl: docs/src/GettingStarted/RunningClimateMachine.md

process will use GPU acceleration. If there are insufficient GPUs (four
in the example above), the `ClimateMachine` processes will share the
GPU resources available.

## Scripts for end-to-end runs, logging and visualization

The `ClimateMachine` [wiki](https://github.com/CliMA/ClimateMachine.jl/wiki)

> details example [SLURM

Reviewer: ```suggestion
contains detailed examples of [SLURM
```

process will use GPU acceleration. If there are insufficient GPUs (four
in the example above), the `ClimateMachine` processes will share the
GPU resources available.

## Scripts for end-to-end runs, logging and visualization

> Our [Wiki](https://github.com/CliMA/ClimateMachine.jl/wiki/Bash-Run-Scripts) details example sbatch scripts that run ClimateMachine.jl, record specified performance metrics and produce basic visualization output. 

Reviewer 2: Formatting suggestion:
```suggestion

The `ClimateMachine` [wiki](https://github.com/CliMA/ClimateMachine.jl/wiki)
details example [SLURM
scripts](https://github.com/CliMA/ClimateMachine.jl/wiki/Bash-Run-Scripts)
that run the `ClimateMachine`, record specified performance metrics and
produce basic visualization output. 
```
